Cloud 2002.0.14 release

hguthrie opened the pull request:
Updates to Cloud documentation for the 2002.0.14 release

- [x] link check
- [x] technical review
- [x] add whatsnew items

whatsnew
Updated the [Release notes for ece-tools version 2002.0.14.](
http://devdocs.magento.com/guides/v2.2/cloud/release-notes/cloud-tools.html#v2002014)
Renamed Local environment setup to Local development. Added new [Docker development](https://devdocs.magento.com/guides/v2.2/cloud/reference/docker-development.html) architecture.
Added [Reset cron jobs](http://devdocs.magento.com/guides/v2.1/cloud/trouble/reset-cron-jobs.html) topic to version 2.1. 
And other minor Cloud guide fixes for broken links and a bad redirect for the ece-tools update.

hostep commented:
@hguthrie or @meker12, just FYI: it looks like there slipped in some merge conflict in the file `guides/v2.3/magento-functional-testing-framework/2.3/getting-started.md`:
```
<<<<<<< HEAD
=======
{% if page.guide_version == "2.2" %}
{% include_relative include/note-2.3-docs.md %}
{% endif %}
 >>>>>>> Removing access-acct files and adding setup folder
```

hguthrie commented:
@hostep This Branch is going through some updates at the moment. Yes, we caught that. Thanks for checking on us! :)

hguthrie commented:
Okay, so the corrections have been made to remove any conflicts. @meker12 Check that the release-related work is intact.

jeff-matthews commented on guides/v2.1/cloud/release-notes/cloud-tools.md:
```diff
@@ -8,8 +8,8 @@ functional_areas:
 ---
-
-<!-- Assigning liquid variables for placeholder values
+<!--MAGECLOUD
```
What is the purpose of this block of liquid tags?

jeff-matthews replied:
The tags and values look like circular references. I don't understand the utility of these. Why not just use plain text? The tags don't appear to offer any convenience. Their presence is confusing.

meker12 replied:
The intent was to show the actual values that cause the deployment error.  Those values are present in the error message. That's why I included them as is.

dshevtsov replied:
@meker12 
I didn't recommend it, I shown how to use it.

jeff-matthews commented on guides/v2.1/cloud/release-notes/cloud-tools.md:
```diff
@@ -25,17 +25,60 @@ php ./vendor/bin/ece-tools list
+   
+   -  <!--MAGECLOUD-2501-->Fixed the cron job workflow in versions 2.1.4 through 2.1.x to unlock frozen cron jobs so that they can be stopped before beginning deployment. Previously, a frozen cron job caused the deployment to stall.
+   
+-  <!--MAGECLOUD-2527-->Changed the format of the `config.php` file generated by the `vendor/bin/ece-tools config:dump` command to use short array syntax and 4-space indentation to comply with Magento coding standards./:q
```
Looks like there are extra random characters at the end of this line?

>... coding standards./:q

hguthrie replied:
Maybe it's an emoji. 😯

jeff-matthews commented on guides/v2.1/cloud/setup/first-time-setup.md:
```diff
@@ -1,60 +1,54 @@
-* [Fastly](https://www.fastly.com/){:target="_blank"} for CDN and caching, used in Staging and Production
-* [Galera Cluster](http://galeracluster.com/){:target="_blank"} for databases, used in Staging and Production (with triple redundancy failover in Production)
+-  **Magento Access Key**  
+    Provides secure access through a 32-character authentication token. The token authenticates between your account, the Magento 2 code repo (repo.magento.com), and any other Git services like GitHub and 3rd party accounts needed for development. You can create multiple Magento access keys. For the workspace setup, you can start with one specific key for your code repository. To generate a key,  ...
```
I think we should be clear that repo.magento.com is . Composer repo. For example:

> The token authenticates between your account, the Magento 2 Composer repo (repo.magento.com)...

jeff-matthews commented on guides/v2.1/cloud/setup/first-time-setup.md:
```diff
@@ -1,60 +1,54 @@
+## Launch a Docker environment {#docker}
 
-Prior to setting up your workspace, you may want to gather the following credentials and accounts:
+You can use the Docker environment to emulates the {{site.data.var.ece}} Integration environment for local development. You need three, essential components: a {{site.data.var.ee}} v2 template, Docker Compose, and {{site.data.var.ece}} `{{site.data.var.ct}}` package.
```
Subject/verb agreement

> You can use the Docker environment to emulate**~s~**

hguthrie commented:
@dshevtsov Just letting you know that we added the whatsnew. We fixed a lot of links. And we are simply waiting for Cloud to release and then we can merge. You are welcome to give it a check while we wait.

dshevtsov commented:
@hguthrie the the __Docker development__ link in the whatsnew returns 404

meker12 commented:
Fixed the link in the comment.
